package/libsolv.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Sun May 28 13:32:15 UTC 2017 - ngompa13@gmail.com
+
+- Enable complex/rich dependencies for CentOS/RHEL 7, matching how
```

Comment:
But the feature is enabled on libsolv in CentOS 7. I checked [the libsolv spec](https://git.centos.org/blob/rpms!libsolv/662e5a792ec1643ca54c2d1ff45c5a4eaca3ed15/SPECS!libsolv.spec#L26) to be sure.